presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+                        "UNION ALL " +
+                        "SELECT UUID() AS uuid, orderkey, orderdate FROM orders",
+                "SELECT 2 * count(*) FROM orders");
+
+        assertUpdate("DROP TABLE test_insert");
+    }
+
+    @Override
+    public void testRenameColumn()
+            throws Exception
+    {
+        // Override because base class error: Must have at least one non-row ID column
+        // Casting to BIGINT -- mvn test: integers... intellij: longs?  weird
+        assertUpdate("CREATE TABLE test_rename_column AS SELECT CAST(123 AS BIGINT) x, 456 a", 1);
+
+        assertUpdate("ALTER TABLE test_rename_column RENAME COLUMN x TO y");
+        MaterializedResult materializedRows = computeActual("SELECT y FROM test_rename_column");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        assertUpdate("ALTER TABLE test_rename_column RENAME COLUMN y TO Z");
+        materializedRows = computeActual("SELECT z FROM test_rename_column");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        assertUpdate("DROP TABLE test_rename_column");
+        assertFalse(queryRunner.tableExists(getSession(), "test_rename_column"));
+    }
+
+    @Override
+    public void testRenameTable()
+            throws Exception
+    {
+        // Override because base class error: Must have at least one non-row ID column
+        // Casting to BIGINT -- mvn test: integers... intellij: longs?  weird
+        assertUpdate("CREATE TABLE test_rename AS SELECT CAST(123 AS BIGINT) x, 456 a", 1);
+
+        assertUpdate("ALTER TABLE test_rename RENAME TO test_rename_new");
+        MaterializedResult materializedRows = computeActual("SELECT x FROM test_rename_new");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        // provide new table name in uppercase
+        assertUpdate("ALTER TABLE test_rename_new RENAME TO TEST_RENAME");
+        materializedRows = computeActual("SELECT x FROM test_rename");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        assertUpdate("DROP TABLE test_rename");
+
+        assertFalse(queryRunner.tableExists(getSession(), "test_rename"));
+        assertFalse(queryRunner.tableExists(getSession(), "test_rename_new"));
+    }
+
+    @Override
+    public void testSymbolAliasing()
+            throws Exception
+    {
+        // Override because base class error: Must specify column mapping property
+        assertUpdate("CREATE TABLE test_symbol_aliasing AS SELECT 1 foo_1, 2 foo_2_4", 1);
+        assertQuery("SELECT foo_1, foo_2_4 FROM test_symbol_aliasing", "SELECT 1, 2");
+        assertUpdate("DROP TABLE test_symbol_aliasing");
+    }
+
+    @Override
+    public void testTableSampleSystem()
+            throws Exception
+    {
+        // Override because base class error: ???
```

Comment:
The Accumulo connector generates one split per tablet, and these test tables are too small to have more than one tablet.  This test returns all 15,000 rows.  If I split the `orders` table in two in the `createAccumuloQueryRunner` function, the test case passes with around 1,800 rows.

Does the sample system only kick in with more than one Presto split?